### ISSUE TITLE
How can I continue training

### ISSUE THREAD

**Issue by indigopyj** [NVlabs/UMR#14]
There is no code for it.

I only used self.model.load_state_dict(model_path) but I think the loss is weird tho:/ the loss is not the value that the loaded model outputs during the previous steps.

Did I miss something?

**Comment by sunshineatnoon** [comment]
Hi, could you please explain more clear what's your task? Do you mean by fine-tune on the pre-trained model?

**Comment by indigopyj** [comment]
> Hi, could you please explain more clear what's your task? Do you mean by fine-tune on the pre-trained model?

No I'm training from scratch now. But actually, when training train_s2.py, the memory space is exploded every ~40 epochs so that the training stops again and again. So I tried to continue training from the step just before it stopped.

**Comment by sunshineatnoon** [comment]
Did you also resume the learning rate?

**Comment by indigopyj** [comment]
> 

I just manually set the learning rate. If it stopped at epoch 20 with lr 0.0002, then I set lr 0.0002 and start from epoch 21.

**Comment by sunshineatnoon** [comment]
I see. That's strange, since if all hyper-parameters are the same, then PyTorch should be able to resume training. Also, what's your GPU and PyTorch version, I didn't see any memory explosion during training on four 16GB GPUs.

**Comment by indigopyj** [comment]
> I see. That's strange, since if all hyper-parameters are the same, then PyTorch should be able to resume training. Also, what's your GPU and PyTorch version, I didn't see any memory explosion during training on four 16GB GPUs.

I used 4 2GB GPUs, this is the reason of memory allocation error. Ok thank you for your answer. I will try my best.
